stable/kubernetes-dashboard/templates/svc.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   ports:
   - name: http
     port: 80
-    targetPort: http
+    targetPort: 80
```

Comment:
ah. just read the spec. it can be. so it was better before. I'll change it back.